resources/default-config.kdl

```diff
@@ -379,6 +379,13 @@ binds {
     XF86AudioMute        allow-when-locked=true { spawn-sh "wpctl set-mute @DEFAULT_AUDIO_SINK@ toggle"; }
     XF86AudioMicMute     allow-when-locked=true { spawn-sh "wpctl set-mute @DEFAULT_AUDIO_SOURCE@ toggle"; }
 
+    // Example media keys mapping using playerctl
+    // This will work with any MPRIS-enabled media player.
+    xF86AudioPlay        allow-when-locked=true { spawn-sh "playerctl play-pause"; }
+    xF86AudioStop        allow-when-locked=true { spawn-sh "playerctl stop"; }
+    xF86AudioPrev        allow-when-locked=true { spawn-sh "playerctl previous"; }
+    xF86AudioNext        allow-when-locked=true { spawn-sh "playerctl next"; }
```

Comment:
```suggestion
    // Example media keys mapping using playerctl.
    // This will work with any MPRIS-enabled media player.
    XF86AudioPlay        allow-when-locked=true { spawn-sh "playerctl play-pause"; }
    XF86AudioStop        allow-when-locked=true { spawn-sh "playerctl stop"; }
    XF86AudioPrev        allow-when-locked=true { spawn-sh "playerctl previous"; }
    XF86AudioNext        allow-when-locked=true { spawn-sh "playerctl next"; }
```

As I asked:

> Just use consistent capitalization with other binds